mgmt sdk release planner bugbash.md

```diff
@@ -0,0 +1,43 @@
+Welcome to Azure Mgmt SDK Release Planner App.The App targets to provide a smooth E2E experience to release Mgmt SDKs.
+- App Link: https://aka.ms/ppe-sdk-workflow
+- Please provide feedbacks by [creating github issue](https://github.com/Azure/azure-sdk-tools/issues/new) with label `Engagement Experience`.
```

Comment:
How about creating bug bash issues in test repo firstly? We can triage them firstly and move them to the formal projects later.

Reply:
Mariana suggested to create issue in azure-sdk-tools directly in last bugbash.